pinot-spi/src/main/java/org/apache/pinot/spi/filesystem/PinotFSFactory.java

```diff
@@ -60,19 +60,19 @@ public static void register(String scheme, String fsClassName, PinotConfiguratio
     }
   }
 
-  public static void init(PinotConfiguration fsConfig) {
+  public static void init(PinotConfiguration config) {
```

Comment:
I see all non-test callers of this api are calling it with fsConfig. Is that expected? Would be good to add javadoc based on what we find.

Reply:
its called with fsFactoryConfig, renamed the variable.